students/ericstreit/lessons/lesson03/assignment/basic_operations.py

```diff
@@ -0,0 +1,121 @@
+"""
+#Lesson03
+#Customer Database Exercise ##
+"""
+
+#!/usr/bin/env python3
+#import stuff here
+from create_db import *
+from db_model import *
```

Comment:
The peewee model set a poor example. Please avoid import *. It makes the name space very messy, and it is hard to figure out where things came from. For example, if something strange happened when running Customer.create, how would you know what file to find it in? What if you hadn't looked at this code in 6 months would you still remember? Yes, of course you can search for it, but better to be explicit.